nl/foreword.md

```diff
@@ -80,13 +80,15 @@ Wij beschouwen deze waarden als de kern van publieke code:
 
 ## Hoe publieke code werkt
 
-Publieke code is open source software bedoeld voor het vervullen van de kerntaken van publieke organisaties. Door gebruik dragen andere organisaties bij aan de software, zodat de ontwikkeling en het onderhoud ervan echte samenwerkingen worden.
+Publieke code is open source software bedoeld voor het vervullen van de kerntaken van publieke organisaties. Wanneer andere organisaties deze open source software gebruiken, krijgen ze de mogelijkheid om hier ook aan bij te dragen. Dit kan door middel van het aandragen van verbeteringen of het aangeven van bugs. Ook kan de organisatie zelf ontwikkelcapaciteit inhuren om features aan de software bij te dragen. Zo wordt de ontwikkeling en het onderhoud ervan een echte samenwerking.
```

Comment:
> zelf ontwikkelcapaciteit inhuren om features aan de software bij te dragen

ik denk dat we daar snel te eenvoudig over denken : ik denk dat we organisaties in ons taalgebruik juist moeten afraden om te denken dat ze eenvoudigweg "ontwikkelcapaciteit" kunnen inhuren om features bij te dragen

Reply:
De uitgebreidere toelichting die nu gegeven wordt komt ook in andere hoofdstukken nog terug. Ik zou het daarom hier weglaten.

Reply:
De uitgebreidere toelichting die nu gegeven wordt komt ook in andere hoofdstukken nog terug. Ik zou het daarom hier weglaten.

Reply:
Ik ben het ermee eens dat het net te kort door de bocht is, maar een uitgebreide aanvulling of andere interpretatie gaat wel weer van de upstream af. Ik zou ofwel een kleine ingreep doen (bijv. "Bij gebruik kunnen andere organisatie bijdragen aan") of iets dergelijks, of terugbrengen naar de upstream om daar een aanpassing te doen.